src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/WasmImportAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices
```

Comment:
Reference assembly update is needed and API review. See https://github.com/dotnet/runtime/issues/93824.